GruntiMaps.WebAPI/Services/GdalConversionService.cs

```diff
@@ -67,6 +67,7 @@ protected override async Task Process()
             // geojson files. Why, you might ask, because tippecanoe can import GeoJSON directly? It's because
             // passing the GeoJSON through ogr2ogr will ensure that the final GeoJSON is in the correct projection
             // and that it should be valid GeoJSON as well.
+            bool converted = false;
```

Comment:
as pointed out by Kai, you can drop the converted bool and just put the await Task.Delay call in an else off the if (gdalMsg != null) part.